packages/storybook-vue/config/main.js

```diff
@@ -8,7 +8,12 @@ module.exports = {
     buildStoriesJson: true,
   },
   stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
-  addons: ['@storybook/addon-links', '@storybook/addon-essentials', 'storybook-addon-pseudo-states'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    'storybook-addon-pseudo-states',
+    '@storybook/addon-a11y',
+  ],
```

Comment:
The addons should also be included in `devDepedencies` of `storybook-vue/package.json`